misc/python/materialize/checks/scenarios_migration.py

```diff
@@ -0,0 +1,43 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+
+from materialize.checks.actions import Action, Initialize, Manipulate, Validate
+from materialize.checks.mzcompose_actions import KillMz, StartMz
+from materialize.checks.scenarios import Scenario
+
+
+class MigrateX86Aarch64(Scenario):
+    """Migrate from x86-64 to aarch64 before the validation, Linux specific since we don't have x86-64 macOS images on Dockerhub"""
+
+    def actions(self) -> list[Action]:
+        return [
+            StartMz(self, platform="linux/amd64"),
+            Initialize(self),
+            Manipulate(self, phase=1),
+            Manipulate(self, phase=2),
```

Comment:
I think the Minipulate No2 should come after the migration. This ensures that DDLs and DMLs are run post-migration as well.

Reply:
I'll try! My main worry was that qemu-static is super slow, so I might have to change some deadlines.